src/components/atoms/Textarea.svelte

```diff
@@ -0,0 +1,27 @@
+<script>
+  export let t_id;
+  export let t_title;
+  export let t_placeholder;
+  export let value;
```

Comment:
i understand why you used this convention, but i found a simple way to do it, please look inputAtom pull request and tell me what do you think

Reply:
I agree with @helbertoro, we must follow the same standard.

Reply:
I reviewed it and I agree with you. I will update it.